pkgs/development/web/postman/darwin.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenvNoCC
+, fetchurl
+, unzip
+, pname
+, version
+, meta
+, ...
+}:
+
+let
+  appName = "Postman.app";
+  dist = {
+    aarch64-darwin = {
+      arch = "arm64";
+      sha256 = "sha256-EtTf17LS18zC3JMbSoyZGGHuIcwGN3Q15XOhVqeh7C4=";
+    };
+
+    x86_64-darwin = {
+      arch = "64";
+      sha256 = "sha256-kTgbqGPgOn5dyjL/IMl3hg2+VUfB+jpPJsqXof8UL+c=";
+    };
+  }."${stdenvNoCC.hostPlatform.system}";
```

Comment:
```suggestion
  }."${stdenvNoCC.hostPlatform.system}" or throw "Unsupported system";
```

Reply:
@SuperSandro2000 Ok I'll add it but may I ask why it is needed because I always wondered? Is meta.platforms not already blocking unsupported systems from building?

Reply:
> Is meta.platforms not already blocking unsupported systems from building?

Yes, but it also needs to eval on other platforms. If this `or` is missing a syntax error would be thrown.

Reply:
@SuperSandro2000 Ah, indeed, the syntax error. Thanks for the explanation👍.